testsuite/integration/basic/pom.xml

```diff
@@ -1136,17 +1448,7 @@
                                 <jboss-maven-dist/>
                                 <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
                             </plugin-options>
-                            <feature-packs>
-                                <feature-pack>
-                                    <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
-                                    <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
-                                    <version>${testsuite.ee.galleon.pack.version}</version>
```

Comment:
I could be wrong but I think the behavior provided by these 3 expressions is being lost.

The goal here of them is to allow control of whether the wildfly-ee or wildfly feature pack is used for provisioning. Standard test runs use the default, which is the wildfly FP. But by setting these props, nightly CI jobs can force the use of wildfly-ee, and we can thus track that that kind of provisioning results in installations that pass the tests.

https://ci.wildfly.org/viewType.html?buildTypeId=WF_MainNightlyJobs_GalleonAltFpLinuxJdk11 does this.

Reply:
Maybe just including another feature pack here will do the trick:

https://github.com/wildfly/wildfly/pull/17153/commits/9f649d7f806acd24d12766ebd89b05932a3be244#diff-100b3a1c5caf72adb576b1434ecc9d3f53b284e79cd07baea4b2fcba616730ffR1137-R1143

If the properties are specifying wildfly-ee then it's redundant as wildfly-test-feature-pack depends on wildfly-ee. But if the props specify wildfly FP then we are bringing that FP into the provisioning.